### ISSUE TITLE
Comments: Form not translated to Dutch

### ISSUE THREAD

**Issue by davoraltman** [Automattic/jetpack#3761]
After I switched my WordPress installation to Dutch and navigated to any of my posts, the comment field wasn't translated.

#### What I expected

The comment form should have been translated to Dutch as per:

https://translate.wordpress.com/projects/wpcom/nl/default/?filters%5Bterm%5D=Notify+me+of+new+comments+via+email&filters%5Buser_login%5D=&filters%5Bstatus%5D=either&filters%5Bview%5D=&filter=Filter&sort%5Bby%5D=priority&sort%5Bhow%5D=desc

#### What happened instead

It wasn't translated and it was still in English - https://cloudup.com/cBVbSw5PbhW

#### Steps to reproduce the issue

- Switch site language to Dutch (Settings -> General and choose Nederlands) and check the comments field on the front end of the site.

Latest version of WordPress. I deactivated all the plugins except Jetpack.

**Comment by jeherve** [comment]
Does the language selection work on WordPress.com sites?

**Comment by davoraltman** [comment]
Yes, it works on WordPress.com sites.

**Comment by barates** [comment]
Hi, same here on my new site (apoplife.nl). Same problem with the Jetpack comments plugin and the Dutch translation....
It also applies to the 'helper' texts shown in the empty fields when entering a new comment.

**Comment by kraftbj** [comment]
Random thought: `WPLANG` is deprecated, replaced with a `WPLANG` option. In our new sync setup, we're syncing `lang_id`, which is an unique option to WP.com that isn't used in Core's implementation of a language selector ( https://github.com/Automattic/jetpack/blob/ad42b94b1d96b95b0e144b74db1f8a1d2cd07f34/sync/class.jetpack-sync-defaults.php#L78 ).

We probably need to sync `WPLANG` and convert that on wpcom to match the lang_id system, assuming that jetpack.wordpress.com comments will respect that for the shadow site.

Haven't fully checked that out though.

**Comment by barates** [comment]
Thank you for the reply, but what does this entail?
Do you think a solution is possible, and if so, could you give an estimate on when?

**Comment by jerrytoet** [comment]
I've the same problem with the ducth language, hopefully someone can post a solution here..

**Comment by kraftbj** [comment]
@barates and @jerrytoet- This would be stuff we need to implement. I don't have a timeline.

Notable that is seems isolated to Dutch.

**Comment by kraftbj** [comment]
I have an idea of what's going on and requested i18n consult in pxLjZ-32h-p2 .

tl;dr: The Jetpack site is sending up `nl` as the language and the process we're using to update the language used is expecting `nl-NL`, if my investigation holds sound.

**Comment by kraftbj** [comment]
We're projecting to have a fix in the next maintenance release of Jetpack which we're expecting in the next few weeks.

**Comment by barates** [comment]
Absolutely great, thanks!

**Comment by jerrytoet** [comment]
Thank you :)

Op donderdag 2 juni 2016 heeft Erwin Barendregt <notifications@github.com>
het volgende geschreven:

> Absolutely great, thanks!
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/Automattic/jetpack/issues/3761#issuecomment-223424413>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ASw4lNF2wEmU_rjmld1Dhe_pBCzRuRkxks5qH0ergaJpZM4IWH5e>
> .
>


-- 
Met vriendelijke groet,
Jerry de Hoon
